README.md

```diff
@@ -14,17 +14,18 @@ Our Android template: **[template](https://github.com/nimblehq/android-templates
 ## Setup
 
 1. Clone or download this repository to your local machine, then extract and open the folder
-2. Install [Kscript](https://github.com/holgerbrandl/kscript#installation)
-3. Run `cd scripts` to get into the scripts directory
-4. Run `kscript new_project.kts` to create a new project with the following arguments:
+2. Install [Kotlin](https://github.com/JetBrains/kotlin) v1.6.21
+3. Install [Kscript](https://github.com/holgerbrandl/kscript#installation) v4.0.3
```

Comment:
How do you think we add the release versions of Kotlin and Kscript? 🤔 Then we could be aware of the latest version on these two repos?

KScript: [![GitHub release](https://img.shields.io/github/release/holgerbrandl/kscript.svg)](https://github.com/holgerbrandl/kscript/releases)

Kotlin: [![Maven Central](https://img.shields.io/maven-central/v/org.jetbrains.kotlin/kotlin-maven-plugin.svg)](https://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.jetbrains.kotlin%22)

Reply:
Hmmm, can we be sure that the script will always work correctly on the latest versions of Kotlin & Kscript? 

The reason we're fixing it here was because of #238, what do you think? 🙏

Reply:
@hoangnguyen92dn I don't think we should keep everything up to date automatically. 🙏🏻

Reply:
@Tuubz @lydiasama That's fine. So we could refer to the fixed versions that the script works.